webapp/advantage/views.py

```diff
@@ -29,11 +30,27 @@
     post_customer_info,
     ensure_purchase_account,
     post_payment_method,
+    invoice_view,
 )
 
 
 session = talisker.requests.get_session()
 
+services = {
+    "canonical-ua": {
+        "short": "ua",
+        "name": "Canonical UA",
```

Comment:
From a branding point of view it's clearly Canonical and if not it's confusingly Canonical on Ubuntu website. Should this just be Ubuntu Advantage?